packages/contracts-bedrock/src/L1/OPStackManager.sol

```diff
@@ -191,6 +211,36 @@ contract OPStackManager is ISemver {
         }
     }
 
+    // / @notice Callable by the OPSM owner to release a set of implementation contracts for a given
```

Comment:
Not using setRelease anymore for initializing the contract. I'm now using the initializer pattern (in 3 parts). 
We may still want this setRelease function to exist though. @mds1